.github/ISSUE_TEMPLATE/task-request.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Task request
+about: Template for development tasks
+title: ''
+labels: ''
```

Comment:
You removed these quotes

```suggestion
labels:
```

In the changes above. Do we need them here? What's the correct syntax?

Reply:
hmm looking into it... it was done through github generator so not sure why they are inconsistent

Reply:
its just YAML underneath so can be either or unless you need to escape something... made them all consistent now